e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction-check-the-information.spec.js

```diff
@@ -89,15 +90,36 @@ context('When fee record correction feature flag is enabled', () => {
       summaryList().should('contain', additionalInfoUserInput);
 
       // The contact email addresses are taken from the bank payment officer team
-      const expectedEmails = BANKS.find((bank) => bank.id === bankId).paymentOfficerTeam.emails;
-      summaryList().should('contain', expectedEmails.join(', '));
+      summaryList().should('contain', expectedBankEmails.join(', '));
     });
 
-    it('should be able to send the record correction request', () => {
-      cy.clickContinueButton();
+    context('when the user clicks the "continue" button', () => {
+      beforeEach(() => {
+        cy.clickContinueButton();
+      });
 
-      cy.visit(`utilisation-reports/${reportId}`);
-      cy.assertText(premiumPaymentsTab.premiumPaymentsTable.status(feeRecordAtToDoStatus.id), 'Record correction sent');
+      it('should be able to send the record correction request', () => {
```

Comment:
I think this should be should send the record correction request and display the ....

Reply:
Thanks for spotting, added the `display the` to the end of the case name - hope this is okay now